pkg/kp/rcstore/store.go

```diff
@@ -82,3 +85,11 @@ func RCUpdateLockPath(rcId fields.ID) (string, error) {
 
 	return path.Join(kp.LOCK_TREE, rcPath, "update"), nil
 }
+
+func IsNotExist(err error) bool {
+	if err == NoReplicationController {
```

Comment:
is this function equivalent to `return err == NoReplicationController` or is there something more subtle going on that prevents this?